_config.yml

```
@@ -55,6 +55,7 @@ minima:
   #social_links:
   #  - { platform: devto,          user_url: "https://dev.to/jekyll" }
   #  - { platform: dribbble,       user_url: "https://dribbble.com/jekyll" }
+  #  - { platform: email,          user_url: "mailto:minima@jekyll.com" }
```

Since this isn't a real email address, I'd prefer to go with a slightly different approach here
```suggestion
  #  - { platform: email,          user_url: "mailto:jekyll_minima@example.com" }
```